tests_mozilla.py

```diff
@@ -583,40 +583,49 @@ def test_get_sld_Mixed_case3(self):
         assert 'example.com' == publicsuffix.get_sld('WwW.example.COM')
 
     def test_get_sld_Leading_dot1(self):
-        assert 'com' == publicsuffix.get_sld('.com')
+        # '.com' -> <empty>.<com> -> None, empty labels are not allowed
+        assert None == publicsuffix.get_sld('.com')
 
     def test_get_sld_Leading_dot2(self):
-        assert 'example' == publicsuffix.get_sld('.example')
+        # '.example' -> <empty>.<example> -> None, empty labels are not allowed
+        assert None == publicsuffix.get_sld('.example')
```

Comment:
the difference here is that example is not in the list. so in strict mode should return None and in non-strict mode would return example --- with the strip

correct?